README.md

```diff
@@ -1,63 +1,49 @@
+3. **Connect to your TV** using [this guide](https://developer.samsung.com/smarttv/develop/getting-started/using-sdk/tv-device.html#:~:text=Connect%20the%20TV%20to%20the%20SDK%3A).
+4. **Create a Samsung certificate** using [this guide](https://developer.samsung.com/smarttv/develop/getting-started/setting-up-sdk/creating-certificates.html).
+5. **Clone/download the repository** and open the `apps` folder of the repository in Tizen Studio by restarting Tizen Studio and changing the workspace.
+6. In the `index.html` file of the Launcher app, change the `IP` variable to the IP of where your debugger will be installed. This could also be the IP of your android device if you plan on using that instead.
+7. Ensure that your TV is selected at the top of Tizen Studio (the dropdown menu).
+8. Right-click the `TizenTube` app and run it as a Tizen web application. Once that is done, do the same for the `Launcher` app.
 
-* First, change the Developer Mode's Host IP to your device's IP.
+After completing these steps, installing apps is complete! You should be able to see the apps on your TV. Now comes the easier part, installing the server or the debugger.
 
-* Download the latest APK compatible with your device's architecute from [here](https://github.com/reisxd/TizenTube/releases/latest) (if unsure, download armeabi-v7a)
+### Option 1: Install on PC/SBC
 
-* Install it
+1. Download NodeJS if you haven't already. Check by running the command `npm -v`.
+2. Clone the repository.
+3. Install modules by running `npm i` in the main folder of the repository.
+4. Install mods modules by running `cd mods` and then running `npm i`.
+5. Build mods by running `npm run build`.
+6. Navigate back to the main folder of the repository by running `cd ..`.
+7. Open `config.json` in your favorite text editor. Change `tvIP` to the IP of your TV. Make sure to leave the `appID` as it is. Change `isTizen3` to true if your TV runs on Tizen 3.
+8. Ensure that SDB is not running by going to Tizen's device manager and disconnecting your TV.
+9. Start the node debugger/server using `node .`.
 
-* After opening the app, change the configuration to your needs
+And now you should have a server running! You should be able to go to your TV's apps, open either the Launcher app or TizenStudio and enjoy adless viewing.
 
-* Press 'Run Server'
+### Option 2: Use The Android App
 
-* Press 'Launch' whenever you want to launch TizenTube
+1. First, change the Developer Mode's Host IP to your device's IP.
+2. Download the latest APK compatible with your device's architecture from [here](https://github.com/reisxd/TizenTube/releases/tag/v1.1.2) (if unsure, download armeabi-v7a).
+3. Install it.
+4. After opening the app, change the configuration to suit your needs. Ensure that you set the `appID` to `Ad6NutHP8l.TizenTube` if it isn't already set. Change the IP to match that of your TV.
```

Comment:
appID is already set to `Ad6NutHP8l.TizenTube` once you install TizenTube Server on Android.

Reply:
I mentioned this because it is difficult for the average user to get this userid again if they accidentally deleted it. I initially thought it was just an arbitrary name but later discovered that it is not. Maybe it would be a good idea to make it uneditable on the Android app?

Reply:
Once you clear the data of the app, it should go back to default configuration.

Reply:
You're correct. Left it in just to avoid confusion.